src/draw/lv_draw_label.c

```diff
@@ -544,7 +545,7 @@ static void draw_letter(lv_draw_unit_t * draw_unit, lv_draw_glyph_dsc_t * dsc,
 
     dsc->letter_coords = &letter_coords;
     dsc->g = &g;
-    cb(draw_unit, dsc, NULL, NULL);
+    cb(t, dsc, NULL, NULL);
```

Comment:
I know this was already there, but could we check this cb before execute it?

Reply:
do you mean like if (cb!= NULL) ?

Reply:
Yes, something like that, if there as guarantee that cb is always valid, that is okay.